backend/de.metas.handlingunits.base/src/main/java/de/metas/handlingunits/impl/CreatePackingInstructionsCommand.java

```diff
@@ -0,0 +1,170 @@
+/*
+ * #%L
+ * de.metas.handlingunits.base
+ * %%
+ * Copyright (C) 2025 metas GmbH
+ * %%
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as
+ * published by the Free Software Foundation, either version 2 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public
+ * License along with this program. If not, see
+ * <http://www.gnu.org/licenses/gpl-2.0.html>.
+ * #L%
+ */
+
+package de.metas.handlingunits.impl;
+
+import de.metas.common.util.time.SystemTime;
+import de.metas.handlingunits.HUItemType;
+import de.metas.handlingunits.IHandlingUnitsDAO;
+import de.metas.handlingunits.model.I_M_HU_PI;
+import de.metas.handlingunits.model.I_M_HU_PI_Item;
+import de.metas.handlingunits.model.I_M_HU_PI_Item_Product;
+import de.metas.handlingunits.model.I_M_HU_PI_Version;
+import de.metas.handlingunits.model.X_M_HU_PI_Version;
+import de.metas.logging.LogManager;
+import de.metas.util.Services;
+import lombok.Builder;
+import lombok.NonNull;
+import org.adempiere.ad.dao.IQueryBL;
+import org.adempiere.model.InterfaceWrapperHelper;
+import org.compiere.model.I_C_UOM_Conversion;
+import org.jetbrains.annotations.NotNull;
+import org.slf4j.Logger;
+
+import java.math.BigDecimal;
+import java.util.List;
+
+import static org.adempiere.model.InterfaceWrapperHelper.saveRecord;
+
+@Builder
+public class CreatePackingInstructionsCommand
```

Comment:
I a resambling command already created for frontend testing: ``de.metas.frontend_testing.masterdata.hu.CreatePackingInstructionsCommand`` . Please, consult with Teo if it makes sense to refactor, to at least have the DAO parts in a single place.

Reply:
no, different parts; but that was as reference

Reply:
agree